packages/devtools_app/lib/src/screens/vm_developer/vm_developer_tools_screen.dart

```diff
@@ -112,8 +112,13 @@ class _VMDeveloperToolsScreenState extends State<VMDeveloperToolsScreenBody>
                 padding: const EdgeInsets.only(
                   left: defaultSpacing,
                 ),
-                child: VMDeveloperToolsScreenBody.views[selectedIndex]
-                    .build(context),
+                child: IndexedStack(
```

Comment:
Will the IndexedStack be necessary after you make the changes we discussed offline to change where the ObjectInspectorViewController is initialized?

Reply:
Probably not, I'll remove it if it's not necessary

Reply:
Keeping the IndexedStack so the full state of the VmDeveloperViews persists (including the ExpansionTile states) when switching through the `NavigationRail`